### ISSUE TITLE
Optimus can get new lane metadata

### ISSUE THREAD

**Issue by kbergin** [HumanCellAtlas/secondary-analysis#96]
New version of function to get optimus lane metadata is needed. Represents changes that need to happen to the input utils python script

AC:
Unit tests that contribute new metadata pass
Updated pipeline can run in Lira in latest schema

____
From other ticket I merged into this one:
[eng] Optimus Adapter WDLs can run Optimus#210
The inputs and outputs of Optimus have changed in master and will be changed by the time there's a new tagged version. We will need to update the adapters to be able to run the pipeline in Lira.

AC:
Pipeline can run in Lira

____

Maybe helpful, notes from ticket before:

We can use this 10x bundle in ingest staging: df04881e-62ce-4f6f-a819-a3b3910431f0

In the bundle, the "dissociation_method" is "fluorescence-activated cell sorting" instead of "10x sequencing" like our registered query expects, but that may just be a placeholder value for the bundle. Mallory would be able to confirm this!

Relevant links: https://github.com/HumanCellAtlas/metadata-schema/pull/320#issuecomment-393957059
https://github.com/HumanCellAtlas/metadata-schema/pull/301

**Comment by samanehsan** [comment]
> In the bundle, the "dissociation_method" is "fluorescence-activated cell sorting" instead of "10x sequencing" like our registered query expects, but that may just be a placeholder value for the bundle. Mallory would be able to confirm this!

After discussing with @malloryfreeberg and @ambrosejcarr we should modify the 10x subscription query that Lira uses to remove the "dissociation_method" field. I remember making a separate issue in elastc to fix that query.

**Comment by malloryfreeberg** [comment]
This makes sense to me. In the future, I think it's only `"library_construction_approach.text": "10X sequencing"` or `"library_construction_approach.ontology": "EFO:0008995"` that you'll want to match on. 

We still need to address the 10X_v2 versus other 10X issue, which is ongoing here: https://github.com/HumanCellAtlas/metadata-schema/issues/426

**Comment by samanehsan** [comment]
The Optimus adapter WDL hasn't been updated lately to work with the latest changes to the Optimus WDL and the submission WDL. The [Optimus workflow](https://github.com/HumanCellAtlas/skylab/blob/master/pipelines/optimus/Optimus.wdl) now expects three arrays of fastq files instead of one array of array of files. In addition, it's outputs have changed. Also the submission WDL no longer has `retry_seconds` and `timeout_seconds` as parameters. 

We'll need to update the adapter WDL before we can test changes to the input utils, but it may make more sense to wait until the pipeline is complete. What do you think @kbergin @ambrosejcarr?

**Comment by samanehsan** [comment]
Here's the branch with the changes to get lane metadata for Optimus that's currently blocked: https://github.com/HumanCellAtlas/pipeline-tools/tree/se-optimus-v6

**Comment by kbergin** [comment]
To follow up with Saman's tag earlier, we re-scoped this ticket, added dependencies and added in other tickets about updating the adapters. We have also de-prioritized this work because we are not about to put in Optimus in Lira anyways and we want to wait until it's settled into a tagged version we can support.

**Comment by dshiga** [comment]
There are no recent 10x bundles following the latest structure and schema. (Most recent ones are from March). This is blocked until we can get a more up to date bundle.

**Comment by dshiga** [comment]
https://github.com/HumanCellAtlas/pipeline-tools/pull/71

**Comment by malloryfreeberg** [comment]
@hewgreen @daniwelter @pnejad @cjvillar @willrockout @zperova Do we have any 10x datasets that we've been able to push to the Data Store in int or staging env?

**Comment by daniwelter** [comment]
I pushed a dummy dataset in int the other day, and I have both a full and cut-down version of the immune cell atlas I can push through. I'm waiting to hear from @dshiga when he needs the next dataset.
